Source/monster.h

```diff
@@ -160,6 +160,8 @@ struct CMonster {
 	const MonsterDataStruct *MData;
 };
 
+struct MissileStruct;

```

Comment:
Mixing in this forward declaration with actual definitions doesn't seem ideal, could this be moved just after the constants/defines at the start of the file? Or maybe just before the function declarations?